### PR TITLE
Mobile styling

### PR DESCRIPTION
**Pull request by mxwnk**

### CODE REVIEW

**Comment by mxwnk** [comment]
Iam sry this PR got a little bit bigger while developing, I hope its still understandable - shame on me 🙈

File: frontend-mobile/lib/widgets/coffee_call.dart [hunk 1]
```diff
@@ -52,28 +52,33 @@ class _CoffeeCallBodyState extends State<CoffeeCallBody> {
...
-              child: const Text('Join'),
-            ),
-            TextButton(
-              onPressed: context.read<CoffeeCallCubit>().leave,
-              child: const Text('Leave'),
+        Padding(
+          padding: const EdgeInsets.all(20),
+          child: Hero(
+            tag: 'coffee-cup',
+            child: Material(
```
Comment (t-animal) [inline frontend-mobile/lib/widgets/coffee_call.dart]: Do we need the `Material` here? My gut feeling tells me, this should be handled by the `MaterialApp `in `main.dart`

File: frontend-mobile/lib/widgets/coffee_call.dart [hunk 2]
```diff
@@ -61,39 +60,55 @@ class _CoffeeCallBodyState extends State<CoffeeCallBody> {
...
-            tag: 'coffee-cup',
-            child: Material(
-              child: ElevatedButton(
-                onPressed: _onCoffeeCupClicked,
-                style: ElevatedButton.styleFrom(
-                  shape: const CircleBorder(),
-                  padding: const EdgeInsets.all(48),
+    return BlocBuilder<CoffeeCallCubit, CoffeeCallState>(
+      builder: (context, state) {
+        return Container(
```
Comment (t-animal) [inline frontend-mobile/lib/widgets/coffee_call.dart]: Could be a padding (only use `Container`s if you need at least two of its properties)

**Review by t-animal — APPROVED** [review]
I'd love a way to directly start a coffee call (e.g. when you're already walking to the coffee machine or already there). Maybe on long press of the button?

**Comment by mxwnk** [comment]
> I'd love a way to directly start a coffee call (e.g. when you're already walking to the coffee machine or already there). Maybe on long press of the button?

see #31

File: frontend-mobile/lib/widgets/coffee_call.dart (hunk 2 above)
Comment (mxwnk) [reply frontend-mobile/lib/widgets/coffee_call.dart]: Done, thanks for the hint, makes sense :)

File: frontend-mobile/lib/widgets/coffee_call.dart (hunk 1 above)
Comment (mxwnk) [reply frontend-mobile/lib/widgets/coffee_call.dart]: Jep, your right! Thats just Copy Pasta

**Pull request merged**